scrapers/HmvMania.yml

```diff
@@ -0,0 +1,50 @@
+name: "HMV Mania"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - hmvmania.com
+    scraper: sceneScraper
+
+sceneByName:
+  action: scrapeXPath
+  queryURL: https://hmvmania.com/?post_type=video&s={}
+  scraper: sceneSearch
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "{url}"
+  scraper: sceneScraper
+
+xPathScrapers:
+  sceneSearch:
+    common:
+      $searchItem: //div[contains(@class,"main-container")]//div[contains(@class,"video-item-horizontal")]
+    scene:
+      Title: $searchItem//h4/text()
+      URL: $searchItem//a[h4]/@href
+      Image: $searchItem//img[not(@class)]/@data-src
+      Details: $searchItem//p/text()
+  sceneScraper:
+    common:
+      $infoNode: //div[@class="main-container"]//div[@class="row"]/div[1]
+    scene:
+      URL: //link[@rel="canonical"]/@href|//meta[@property="og:url"]/@content
+      Image: //meta[@property="og:image"]/@content
+      Title:
+        selector: $infoNode//h1[@class="video-entry-title"]/text()
```

Comment:
I thought about it.
I submitted the PR the way I settled on using it myself.
I could've used Studio field instead of Director like I do for MMD videos, but for HMV it seems to make less sense.
Director field is a new addition and has a low visibility, buried inside the details page, so it's like the opposite of elevating it to the Studio rank.
When there is no clear best approach - I decide on keeping more flexibility.

I feel like `stash` is terribly specific in its design, so I can't even hope for a common practice or guideline for how to deal with "weird" stuff like HMV/PMV/CH/FH/MMD.
I wish it had an abstract model that could be adopted to various ontologies, instead of being "one fits all, but not really" solution.

Reply:
You can easily rename the fields to mean something different via `Custom localization` in Settings > Interface, but yeah, it was designed with mainstream scenes in mind first.

Reply:
After some consideration, I decided to switch to the Studio field instead of Director and clean the now redundant maker name from title and details.
Some prominent authors can have online presence, logo/avatar etc. So, it might be not a bad idea to treat them the same as MMD authors after all.